### DISCUSSION TITLE
Integrate this repo is oss-fuzz

### DISCUSSION THREAD

**Discussion by thealberto** [libpng/libpng#8] [General]
Hi all,
I have spent some time improving the `oss-fuzz` coverage on the main `libpng` repo and I had already some promising results. Would you consider worth to try to integrate this repo into `oss-fuzz` so I could work on new fuzzers and possibly to identify new vulnerabilities? More than happy to help to fix it if possible.

Thanks

**Comment by ioquatix** [comment]
Yes, absolutely, sounds like a great idea! I'm all for fuzzing!

**Comment by thealberto** [comment]
Hey, maybe now it would be better to just use the original repo?

**Reply by ioquatix** [reply]
Why is that?
